core/src/main/resources/reference.conf

```diff
@@ -268,6 +268,21 @@ datastax-java-driver {
     warn-on-init-error = true
   }
 
+  # Whether to schedule reconnection attempts if the driver fails to reach any contact point on the
```

Comment:
I think it would be less vague if it said "... if the driver fails to reach all contact points ... " to imply that it will try all contact points before reconnecting.